packages/integrations/tailwind/README.md

```diff
@@ -61,6 +61,65 @@ export default defineConfig({
 });
 ```
 
+Then, create a `tailwind.config.js` file in your project's root directory. You can use the following command to generate a basic configuration file for you:
+
+```sh
+npx tailwindcss init
+```
+
+Add this basic configuration to your `tailwind.config.js` file:
+
+```js ins={4} "content: ['./src/**/*.{astro,js,ts,tsx,md,mdx}']"
+// tailwind.config.js
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{astro,js,ts,tsx,md,mdx}"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
+```
+
+:::note
+If you're using any other UI framework, remember to add the extension to the content array. For example, if you're using Vue, you'll need to add `./src/**/*.vue` to the content array.
+:::
+
+Add a directory `styles` to your `src` directory, and create a file called `global.css` inside of it. This is where you can add your own custom CSS styles or directives. For example, you can add a `@layer` directive to add your own custom styles to the `base` layer:
```

Comment:
> I think the base.css is optional because it's already injected by the integration but I'd like someone to confirm

Yeah I think we can remove this part here. The `applyBaseStyles` option covers this, and already has some docs that cover needing to import the base style manually.